pkg/build/node/node.go

```diff
@@ -355,6 +355,12 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 		return err
 	}
 
+	// Create "images" subdir.
+	imagesDir := path.Join(dir, "bits", "images")
```

Comment:
at https://github.com/kubernetes-sigs/kind/pull/104/files#diff-7033e7fa5be88fc307b556fb3888ef42R374 we should use `pullTo := path.Join(imagesDir, image)` now that we have this value, so we can ensure they stay consistent.

Reply:
Done